.github/workflows/build-pkg.yml

```diff
@@ -101,6 +104,40 @@ jobs:
                           --package-path Homebrew.pkg Homebrew-${{ steps.print-version.outputs.version }}.pkg
                           --sign "${PKG_APPLE_DEVELOPER_TEAM_ID}"
 
+      - name: Remove existing Homebrew installations
+        run: |
+          sudo rm -rf brew /{usr/local,opt/homebrew}/{Cellar,Caskroom,Library/Taps}
```

Comment:
`/usr/local/Library/Taps` does not exist; it should be `/usr/local/Homebrew/Library/Taps`.

Reply:
@carlocab good catch, fixed.